_posts/2025-11-21-verilator-extension.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: Update on Fault-Injection with Verilator
+author: Jonathan Schröter
+date: 2025-11-24
+---
+As part of our ongoing efforts to improve tooling for hardware verification and software–hardware co-design, we have extended Verilator with support for fault-injection mechanisms.  
+Below is a concise overview of the changes introduced and how to use the extension.
+
+If you’re looking for background on the motivation behind this feature, please refer to "Fault-injection support with the Verilator simulation tool" in the [Isolde project overview](https://aemy.cs.hm.edu/projects/isolde).
+
+# What's new
```

Comment:
What is this "new" relative to? I assume this is about what is being added to Verilator?